### ISSUE TITLE
Bind response - 0x8030

### ISSUE THREAD

**Issue by pipiche38** [fairecasoimeme/ZiGate#122]

**Comment by pipiche38** [comment]
Would it be possible to have the source address in the bind response ?
either the IEEE or Short address would be welcome

**Comment by ISO-B** [comment]
Adding following lines
```
ZNC_BUF_U8_UPD ( &au8LinkTxBuffer [u16Length], psStackEvent->uEvent.sApsDataIndEvent.u8SrcEndpoint, u16Length );
ZNC_BUF_U8_UPD ( &au8LinkTxBuffer [u16Length], psStackEvent->uEvent.sApsDataIndEvent.u8SrcAddrMode, u16Length );
```
after
https://github.com/fairecasoimeme/ZiGate/blob/d18f5e7e82f0660f909d144d217198f0468be072/Module%20Radio/Firmware/src/ZiGate/Source/ZigbeeNodeControlBridge/app_general_events_handler.c#L852

should do the job. I can't see any reason why this couldn't be added.

**Comment by pipiche38** [comment]
@ISO-B Are you in keybase.io ? Would like to chat in MP

**Comment by pipiche38** [comment]
@ISO-B , @fairecasoimeme  
it seems that it has not been implemented in 3.0f , correct ?

**Comment by pipiche38** [comment]
@ISO-B if you can make this one in 3.10, this would be great!

**Comment by ISO-B** [comment]
This will be in next release version

**Comment by pipiche38** [comment]
Just check your PR. Is there a way to get also the Cluster on which the bind is made ?

**Comment by doudz** [comment]
Doc not up to date

**Comment by pipiche38** [comment]
For me the 0x8030 (and true also for 0x8031) are incomplete, as we also need to know what is the cluster for this bind

**Comment by ISO-B** [comment]
Correct response for both commands is:
```
<Sequence number: uint8_t>
<Status: uint8_t>
<SrcEndpoint: uint8_t>
<SrcAddrMode: uint8_t>
<SrcAddr: uint16_t>
<ClusterId: uint8_t>
```

**Comment by pipiche38** [comment]
Please do check what is really happening. 
The data structure is probably correct. I' not discussing it, but the ClusterId value is set to the ShortAddress !

And SrcAddrMode Type should be based on SrcAddrMode

**Comment by pipiche38** [comment]
Will be fixed in 3.1b

<Sequence number: uint8_t>
<Status: uint8_t>
<SrcAddrMode: uint8_t>
<SrcAddr: uint16_t>

**Comment by doudz** [comment]
This issue should be reopened because cluster ID is not available on 3.1b and 3.1c
see https://github.com/doudz/zigate/issues/170
